### PR TITLE
Make heat-map zoom intensities customizable

### PR DESCRIPTION
**Pull request by mangerlahn**
This small PR is based on #143 , adding customizable zoom intensities to the heat-map.

### CODE REVIEW

**Comment by mangerlahn** [comment]
I signed it!

File: src/Heatmap/GMUHeatmapTileLayer.m [hunk 1]
```diff
@@ -24,318 +24,331 @@
...
 }
 @end
 
 @implementation GMUHeatmapTileCreationData
 @end
 
 @implementation GMUHeatmapTileLayer {
-  BOOL _dirty;
-  GMUHeatmapTileCreationData *_data;
+    BOOL _dirty;
```
Comment (domesticmouse) [inline src/Heatmap/GMUHeatmapTileLayer.m]: I'm curious about this whitespace. It looks like you have re-formatted the majority of this file away from Google style. 

/cc @dion-loetscher any thoughts on adding a clang format to this project?

Comment (dion-loetscher) [reply src/Heatmap/GMUHeatmapTileLayer.m]: @domesticmouse I think adding a clang format is a good idea. 

I think we should hold off on approving this PR until we have added clang format. Then we can revisit this PR after it's been clang formatted.

**Comment by mangerlahn** [comment]
Thanks for the review! Although I agree that clang format would be appreciated here, converting this file was more Xcode's decision than mine. I fixed this for now, in case its the only reason this might not get merged.

**Pull request merged**